projects/astc-encoder/astc_load_decompress_image_fuzzer.cc

```diff
@@ -95,26 +74,29 @@ extern "C" int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size) {
 
   astcenc_config config{};
   astcenc_preset preset = ASTCENC_PRE_FAST;
-  error = astcenc_config_init(profile, image_comp.block_x, image_comp.block_y,
-                              image_comp.block_z, preset, 0, config);
-  if (error)
+  if (astcenc_config_init(profile, image_comp.block_x, image_comp.block_y,
+                          image_comp.block_z, preset, 0,
+                          config) != ASTCENC_SUCCESS)
     return 0;
 
+  astcenc_swizzle default_swizzle{/*astcenc_swz r=*/ASTCENC_SWZ_R,
+                                  /*astcenc_swz g=*/ASTCENC_SWZ_G,
+                                  /*astcenc_swz b=*/ASTCENC_SWZ_B,
+                                  /*astcenc_swz a=*/ASTCENC_SWZ_A};
```

Comment:
Nit: I don't think you need the clarifying comments here, since I think R-G-B-A is a common enough pattern, but this is definitely a readability judgment call.

Also, you don't need to specify types in these clarifying comments.